docs/user-guide/cli.md

```diff
@@ -12,10 +12,12 @@ npm install stylelint --save-dev
 
 ## Usage
 
-`stylelint --help` prints the CLI documentation.
+`npm run stylelint --help` prints the CLI documentation.
```

Comment:
This will not work unless the user has added a `stylelint` scrip in their `package.json` file

Perhaps `npx run stylelint --help`, but again as highlighted in the issue, it's difficult to assume how a user uses stylelint...

I think i'd prefer no change on this line, for the reasons outlined here and in the issue

Reply:
Today I learned. It works with `yarn` and I'm so used to almost everything being the same across the CLI interfaces. Sorry for the trouble and thanks for catching it!